schema/User.re

```diff
@@ -0,0 +1,16 @@
+type role =
```

Comment:
nice we can even experiment after having Model + entity GraphQL schema in one file after

Reply:
yeah nice one, probably will be easier to pass types reference, actually by what I was seeing is how they do with elixir

Reply:
makes totally sense indeed